core/tests/puppeteer-acceptance-tests/utilities/common/test-constants.ts

```diff
@@ -235,4 +235,8 @@ export default {
     'Art and Design',
   ],
   DEFAULT_SPEC_TIMEOUT_MSECS: 300000,
+  // Video Recording Path in relative to root of oppia folder.
```

Comment:
Not sure what this comment is saying.  Maybe: "The video recording path below is relative to the oppia/ folder's root path."

Reply:
Done!